Documentation/ceph-prerequisites.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Prerequisites
+weight: 2010
+indent: true
+---
+
+# Ceph Prerequisites
+
+## LVM package
+
+Ceph OSDs have a dependency on LVM in the following scenarios:
+- OSDs are created on raw devices or partitions
+- If encryption is enabled (`encryptedDevice: true` in the cluster CR)
+- A `metadata` device is specified
+
+LVM is not required for OSDs in these scenarios:
+- Creating OSDs on PVCs using the `storageClassDeviceSets`
+
+If LVM is required for your scenario, LVM needs to be available on the hosts where OSDs will be running.
+Some Linux distributions do not ship with the `lvm2` package. This package is required on all storage nodes in your k8s cluster to run Ceph OSDs.
+Without this package Rook will be able to successfully create the Ceph OSDs. However, when a node is rebooted the OSD pods
+running on the restarted node will **fail to start**. Please install LVM using your Linux distribution's package manager. For example:
+
+```console
+# Centos
```

Comment:
What about moving `Centos` outside of the console section and rename it `For Red Hat based systems`.